src/fights.ts

```diff
@@ -261,14 +261,15 @@ function getEmbezzlerFight(): EmbezzlerFight | null {
     if (fight.available()) return fight;
   }
   const potential = embezzlerCount();
-
+  const averageEmbezzlerNet = ((baseMeat + 750) * meatDropModifier()) / 100;
   if (potential > 0) {
     print(`You have the following embezzler-sources untapped right now:`, "blue");
     embezzlerSources
       .filter((source) => !source.available() && source.potential() > 0)
       .map((source) => `${source.potential()} from ${source.name}`)
       .forEach((text) => print(text, "blue"));
     if (
+      Math.floor((potential * averageEmbezzlerNet) / 50000) > 0 &&
```

Comment:
This feels like a profoundly weird way to do the math on this--is there a reason you've chosen to execute it this way?

Reply:
I swear it made sense in my head at that the time. We can just do potential * averageEmbezzlerNet >50000 I suppose

Reply:
Make it so and I'll push it